packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -119,76 +124,111 @@ export class GCTelemetryTracker {
 			return false;
 		}
 
+		// Non-tombstone events are logged once per event per node. A unique id is generated by joining
```

Comment:
Maybe add a note in the function's doc comment that this doesn't apply to Tombstone events, they are always logged.